### ISSUE TITLE
FTB infinity- unable to load world

### ISSUE THREAD

**Issue by FlameHazard** [thraaawn/CompactMachines#109]
I have recently started a new FTB infinity world and decided I want Compact Machines in the pack. I downloaded it striaght from the website (The curse forge one) and added it to the pack using the "add mod" button. Now when I try to load a world or create a new one it kicks me back to the title screen. I have tried everything from restarting my computer to re installing the mod. I hope you can help.

http://pastebin.com/embed_js.php?i=5qP2dBNB (sorry the first part of the log is the launch, it's lengthy).

**Comment by thraaawn** [comment]
```java.lang.IllegalArgumentException: Failed to register dimension for id 2, One is already registered```

Change the dimensionId in the CompactMachines config.

**Comment by FlameHazard** [comment]
I have changed the ID to all sorts of numbers. I these are the ones I have:
DIM1
DIM-1
DIM2
DIM-7
DIM-19
DIM-34
DIM-37
DIM-38
DIM-42
DIM-100
DIM-112

Plus mystcraft dimensions 1 through 12. I've changed the CM dimension to all sorts of numbers and I still get the same error. I'm saving my config every time I change it and I've made sure there are no other dimensions numbered the one I put. What should I set it to?
